Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -0,0 +1,109 @@
+#import "SentryTimeToDisplayTracker.h"
+#import "SentryCurrentDate.h"
+#import "SentrySpan.h"
+#import "SentrySpanContext.h"
+#import "SentrySpanId.h"
+#import "SentrySpanOperations.h"
+#import "SentrySwift.h"
+
+#if SENTRY_HAS_UIKIT
```

Comment:
`l`: can be moved above imports because all @interface and @implementation is guarded by this